docs/tutorials/run-alpaca-eval.md

```diff
@@ -0,0 +1,107 @@
+# Running Alpaca Eval with Marin
+
+This tutorial shows how to configure and launch the Alpaca evaluation pipeline in Marin, using the `evaluate_alpaca_eval` helper from `experiments/evals/evals.py`. For an overview of evaluation concepts, see [Evaluation Overview](../explanations/evaluation.md).
+
+## Prerequisites
+
+- A trained model checkpoint in Hugging Face (HF) format.
```

Comment:
also this doc is pretty clearly limited to TPU right now, and probably TPU v6e?

Reply:
Yeah, that's true for all eval except for Levanter evals rn. I think we'll need a pretty significant refactor to evals post-launch to add GPU support (which is vaguely ironic since GPU is better supported by the underlying lib)